src/app/components/CopyToClipboard/index.tsx

```diff
@@ -47,7 +48,15 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = ({ label, value }) => {
   }, [])
 
   return (
-    <Tooltip arrow onOpen={hideTooltip} open={isCopied} placement="right" title={t('clipboard.success')}>
```

Comment:
Title is inaccurate - we already had a default animation

| Before | After |
| --- | --- |
|  ![Recording 2023-02-27 at 19 24 26](https://user-images.githubusercontent.com/3758846/221651546-3a45e8d2-6bc1-4e7a-b132-464b945001fc.gif) | ![r](https://user-images.githubusercontent.com/3758846/221656116-27fedff1-25e0-4226-b5f1-53f4e226447b.gif) |

Reply:
And this is now inconsistent with other tooltips

![Recording 2023-02-27 at 19 48 43](https://user-images.githubusercontent.com/3758846/221655443-d19eccf7-60ec-4317-9aa3-d9165e5f8bfd.gif)

Reply:
Indeed, so the default animation was Grow, which we are replacing with Fade now, as specified by @donouwens.
I have updated the title of the PR to reflect that.
Indeed, there are other places where we also have similar tooltips, I am guessing that we should replace all of them, too, but I have asked @donouwens to confirm before proceeding.

Reply:
I have already updated the code to use `Fade` everywhere. 
And according to @donouwens :

> My preference is Fade - as it’s a bit calmer.

Reply:
> I have already updated the code to use Fade everywhere.

I mean, all the _two_ locations.

Reply:
Timings aren't consistent :D 

perhaps defaultTheme is a better place for this
```ts
    MuiTooltip: {
      defaultProps: {
        TransitionComponent: Fade,
        TransitionProps: { timeout: 600 },
      },
```

Reply:
Indeed. Moved in there.